### PR TITLE
[AIFBR-8323] CORS policy backup/restore for each bucket

### PR DESCRIPTION
**Pull request by vivekjhauipath**

### CODE REVIEW

File: platform/onebox/backup_and_restore/ceph/import.sh [hunk 1]
```diff
@@ -45,23 +45,39 @@ function list_buckets() {
...
+  echo "$green $(date) Finsihed sync of object storage to local disk for bucket ${BUCKET_NAME} $default"
+}
+
+function update_cors_policy() {
+  BUCKET_NAME=${1}
+  if [ ! -f "${FOLDER}${BUCKET_NAME}-cors.json" ]; then
+    echo "$red $(date) ${FOLDER}${BUCKET_NAME}-cors.json file does not exist, Please check ... Skipping cors creation $default"
+    return
+  fi
+  aws --endpoint-url $AWS_ENDPOINT --no-verify-ssl s3api put-bucket-cors --bucket ${BUCKET_NAME} --cors-configuration file://${FOLDER}${BUCKET_NAME}-cors.json
```
Comment (rajivml) [inline platform/onebox/backup_and_restore/ceph/import.sh:70]: cors policy is getting exported here, so the method name should be export_cors_policy ?

File: platform/onebox/backup_and_restore/ceph/export.sh [hunk 2]
```diff
@@ -35,28 +35,42 @@ function initialize_variables() {
...
+function get_cors_policy() {
+  BUCKET_NAME=${1}
+  aws --endpoint-url $AWS_ENDPOINT --no-verify-ssl s3api get-bucket-cors --bucket ${BUCKET_NAME} > ${FOLDER}${BUCKET_NAME}-cors.json
+}
+
+function download_blob() {
+  BUCKET_NAME=${1}
+  echo "$green $(date) Starting sync of object storage to local disk for bucket ${BUCKET_NAME} $default"
+  mkdir -p ${FOLDER}${BUCKET_NAME}
+  aws s3 --endpoint-url $AWS_ENDPOINT --no-verify-ssl sync s3://${BUCKET_NAME} ${FOLDER}${BUCKET_NAME} --delete
```
Comment (rajivml) [inline platform/onebox/backup_and_restore/ceph/export.sh:55]: are we checking anywhere if "aws s3" CLI is installed ?

**Review by rajivml — COMMENTED** [review]
clarifications needed

File: platform/onebox/backup_and_restore/ceph/export.sh (hunk 2 above)
Comment (vivekjhauipath) [reply platform/onebox/backup_and_restore/ceph/export.sh:55]: yes, in pre-req.

File: platform/onebox/backup_and_restore/ceph/import.sh (hunk 1 above)
Comment (vivekjhauipath) [reply platform/onebox/backup_and_restore/ceph/import.sh:70]: It's not exported, but getting added to bucket here. Hence name is update_cors_policy.

**Comment by vivekjhauipath** [comment]
> clarifications needed

Clarified on those. Checking for s3 is already done as part of earlier merge.

**Review by mohitseth-nitrr — APPROVED** [review]
LGTM

**Review by rajivml — APPROVED** [review]

**Pull request merged**